release/release_test.go

```diff
@@ -50,3 +52,42 @@ func (s *ReleaseTestSuite) TestOverride(c *C) {
 	c.Check(release.String(), Equals, "10.06-personal")
 	c.Check(release.Get(), DeepEquals, rel)
 }
+
+func makeMockLsbRelease(c *C) string {
+	// FIXME: use AddCleanup here once available so that we
+	//        can do release.SetLsbReleasePath() here directly
+	mockLsbRelease := filepath.Join(c.MkDir(), "mock-lsb-release")
+	s := `
+DISTRIB_ID=Ubuntu
+DISTRIB_RELEASE=18.09
+DISTRIB_CODENAME=awsome
+DISTRIB_DESCRIPTION=I'm not real!
+`
+	err := ioutil.WriteFile(mockLsbRelease, []byte(s), 0644)
+	c.Assert(err, IsNil)
+
+	return mockLsbRelease
+}
+
+func (a *ReleaseTestSuite) TestReadLsb(c *C) {
+	realLsbRelease := release.LsbReleasePath()
+	defer func() { release.SetLsbReleasePath(realLsbRelease) }()
```

Comment:
Thanks, excellent suggestion and a pattern I will keep in mind for the future! Fixed in git.